README.md

```diff
@@ -18,7 +18,6 @@ Your choice of server software can make a huge difference in performance and api
 
 Recommended top picks:
 * [Paper](https://github.com/PaperMC/Paper) - The most popular server software that aims to improve performance while fixing gameplay and mechanics inconsistencies.
-* [Tuinity](https://github.com/Spottedleaf/Tuinity) - Paper fork that includes even more high-performance patches.
 * [Airplane](https://github.com/Technove/Airplane) - Tuinity fork that aims to further improve server performance.
```

Comment:
how come tuinity for is real when tuinity is not real

Reply:
because merge conflicts